incubator/hnc/config/default/kustomization.yaml

```diff
@@ -17,7 +17,10 @@ bases:
 - ../rbac
 - ../manager
 - ../webhook
-- ../certmanager
+# Add cert management. Only one of the ../internalcert and the
+# ../certmanager should be enabled.
```

Comment:
Also mention where `--enable-internal-cert-management` should be added or removed.

Reply:
Good point! Done, thanks!